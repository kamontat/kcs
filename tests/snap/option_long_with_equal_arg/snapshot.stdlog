[DBG] init.options         reuse definitions from cache file '$TMPDIR/kcs/libs-options/options-definitions.txt'
[DBG] hook-init.options    options definition: |-l|:LAND:NV::;|-f|--flower|:FLOWER:NV::;|-s|--sky|:SKY:RS::;|-w|--water|:WATER:OS::;|-r|--rock|:ROCK:OS:paper:;|-e|--ex|--extra|:EXTRA:NV:paper:;|--q|:QUE:NV:paper:;
[DBG] hook-init.options    parsing long options '--rock'
[DBG] checker.options      inline mode; never parse argument for '--rock'
[DBG] export.options       export '_KCS_OPT_ROCK_VALUE'='water'
