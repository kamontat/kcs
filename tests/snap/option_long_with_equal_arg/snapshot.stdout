Direct Arguments: 1 [--rock=water]
Parsed Arguments: 0 []
Rock: water
